src/actionlist/action-list-tree.scss

```diff
@@ -21,14 +33,104 @@
   }
 
   .ActionList-item {
+    // start: copy from ActionList proper- backwards compatible for treeview with different markup structure
+    &[aria-expanded] {
+      .ActionList--subGroup {
+        @media screen and (prefers-reduced-motion: no-preference) {
+          transition: opacity 160ms cubic-bezier(0.25, 1, 0.5, 1), transform 160ms cubic-bezier(0.25, 1, 0.5, 1);
+        }
+
+        .ActionList-content {
+          padding-left: $spacer-4;
+        }
+      }
+
+      // has 16px leading visual
+      .ActionList-content--visual16 + .ActionList--subGroup {
+        .ActionList-content {
+          padding-left: $spacer-5;
+        }
+      }
+
+      // has 20px leading visual
+      .ActionList-content--visual20 + .ActionList--subGroup {
+        .ActionList-content {
+          padding-left: $spacer-2 * 4.5; // 36px
+        }
+      }
+
+      // has 24px leading visual
+      .ActionList-content--visual24 + .ActionList--subGroup {
+        .ActionList-content {
+          padding-left: $spacer-6;
+        }
+      }
+    }
+
+    &[aria-expanded='true'] {
+      .ActionList-item-collapseIcon {
+        transition: transform 120ms linear;
+        transform: scaleY(-1);
+      }
+
+      .ActionList--subGroup {
+        height: auto;
+        overflow: visible;
+        visibility: visible;
+        opacity: 1;
+        transform: translateY(0);
+      }
+
+      &.ActionList-item--hasActiveSubItem {
+        > .ActionList-content > .ActionList-item-label {
+          font-weight: $font-weight-bold;
```

Comment:
Does this bold font get applied to PR-like TreeView as well?

Reply:
Oh good catch! I have a task to _remove_ this for tree-view actually, so I can handle that here :)